tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
```

Comment:
seems tox still has value for virtualenv management, even if it is set up to target only one Python version

Reply:
I think there are three options for testing (travis-ci, Alfred, tox), and all of them are only testing `py35` right now. We could expand these to include `py36`, `py37`, `py38`. Adding some of these soon might be a good idea to lessen the risk of surprises.